docs/additional/interesting_aspects_de.md

```diff
@@ -0,0 +1,248 @@
+- Überprüfen der UID & GID für Dateien (startup.sh, resources, Volumes, Schreib-abhängige Pfade)
+- Erreichbarkeit der Software: TCP-Health-Checks
+
+Für die Integration in eine Pipeline empfehlen wir den Einsatz der `ecoSystem.verify()` Methode aus der [dogu-build-lib][dogu-build-lib].  
+Ein beispielhafter Aufruf sieht in einem Jenkinsfile wie folgt aus:
+
+```groovy
+stage('Verify') {
+    ecoSystem.verify("/dogu")
+}
+```
+
+Dieser ruft intern den Befehl `cesapp verify` auf, welcher die Goss-Tests ausführt.
+
+[goss]: https://github.com/goss-org/goss
+[dogu-build-lib]: https://github.com/cloudogu/dogu-build-lib
+
+## Dokumentation
+
+Jedes Tool sollte eine sinnvolle Dokumentation über verwendete Features besitzen, die während der Benutzung oder Support unterstützt.
+Um eine einfache Integration in unsere vorhandene Dokumentations-Infrastruktur zu ermöglichen, sollte die Dokumentation in einem im Repository-Root befindlichen Order `/docs` abgelegt werden.
+
+Standardmäßig umfasst die Doku folgende Dinge:
+- Features von dem Dogu
+- Installation des Dogus
+- Konfiguration des Dogus
+- Entwicklung am Dogu
+- Readme
+- Changelog
+
+### Lizenz
+
+Die Nutzungslizenz des Dogus sollte deutlich hervorgehen.
+Diese kann zum Beispiel in der Readme angegeben werden.
+
+### Readme
+
+Eine Readme ist sowohl für eigene Entwicklungen als auch im Partnerkontext mit externen Entwicklern sinnvoll.  
+Bei öffentlichen Repositories sollte jedoch ein höherer Maßstab an einer Readme angelegt werden.
+
+Eine Readme-Datei liegt im Repository-Root und beschreibt knapp Hinweise zum generellen Zweck des Dogus.
+Dies kann in Form eines Quickstart-Guides sein, indem zum Beispiel beschrieben ist wie man eine lokale Entwicklungsumgebung einrichtet.
+Es sollte außerdem ein Verweis auf den Dokumentationsordner beinhalten und kann darüber hinaus auch auf spezielle Seiten der Dokumentation verweisen.
+Zum Schluss ist es zu empfehlen benötigte Ressourcen für den Gebrauch und einen Verantwortlichen des Dogus anzugeben.
+
+### Changelog
+
+Changelogs werden für Menschen geschrieben, die Neuigkeiten in neueren Releases verstehen möchten.
+
+Das Format des Changelogs sollte nach **[keep a changelog](https://keepachangelog.com)** angelegt sein.
+
+Neue Änderungen werden stets in die oberste Sektion `## [Unreleased]` hinzugefügt.  
+Bei einem neuen Release wandern diese Änderungen in eine neue Sektion, die die entsprechende Versionsnummer enthält.
+
+Bei allen Änderungen sollten Verweise auf die Vorgänge des verwendeten Issue-Trackers hinzugefügt werden.  
+Handelt es sich außerdem um komplexere Vorgänge, ist zusätzlich ein Verweis zu einer Dokumentation sinnvoll.
+
+### Systemanforderungen
+
+Die Systemanforderungen eines Dogus (CPU, RAM, Speicher) können entweder in der Dokumentation des Dogus festgehalten werden
+oder sollten Cloudogu auf andere Weise direkt mitgeteilt werden (z.B. im Falle eines privaten Repositories).
```

Comment:
Phrases that contain whitespcace in its unabbreviated form keep the whitespace in its abbreviation.

```suggestion
oder sollten Cloudogu auf andere Weise direkt mitgeteilt werden (z. B. im Falle eines privaten Repositories).
```